### PR TITLE
Feature/oepcis 521 fixing issues associated ilmd marshalling

### PR DESCRIPTION
**Pull request by Aravinda93**
This PR will fix the issues associated with namespaces during the document/event conversion from XML <-> JSON that were caused due to recent changes. Kindly request you to review and approve the PR.

### CODE REVIEW

File: src/main/java/io/openepcis/convert/collector/JsonEPCISEventCollector.java [hunk 1]
```diff
@@ -109,6 +113,9 @@ public void start(Map<String, String> context) {
               });
       jsonGenerator.writeEndArray();
 
+      // Reset the modified namespaces
+      defaultJsonSchemaNamespaceURIResolver.getInstance().resetModifiedNamespaces();
```
Comment (sboeckelmann) [inline src/main/java/io/openepcis/convert/collector/JsonEPCISEventCollector.java:117]: why getInstance() ? don't you have a private instance?

File: src/main/java/io/openepcis/convert/collector/JsonEPCISEventCollector.java [hunk 2]
```diff
@@ -173,11 +180,12 @@ public void startSingleEvent(Map<String, String> context) {
       jsonGenerator.writeString(EPCISVersion.getDefaultJSONContext());
 
       // Modify the Namespaces so trailing / or : is added and default values are removed
-      DefaultJsonSchemaNamespaceURIResolver.getInstance().modifyEventNamespaces();
+      defaultJsonSchemaNamespaceURIResolver.getInstance().modifyEventNamespaces();
```
Comment (sboeckelmann) [inline src/main/java/io/openepcis/convert/collector/JsonEPCISEventCollector.java:183]: why getInstance() ? don't you have a private instance?

File: src/main/java/io/openepcis/convert/collector/JsonEPCISEventCollector.java [hunk 3]
```diff
@@ -173,11 +180,12 @@ public void startSingleEvent(Map<String, String> context) {
...
 
       // Get all the stored namespaces from jsonNamespaces
-      DefaultJsonSchemaNamespaceURIResolver.getInstance()
-          .getEventNamespaces()
+      defaultJsonSchemaNamespaceURIResolver
+          .getInstance()
```
Comment (sboeckelmann) [inline src/main/java/io/openepcis/convert/collector/JsonEPCISEventCollector.java:187]: why getInstance() ? don't you have a private instance?

File: src/main/java/io/openepcis/convert/collector/JsonEPCISEventCollector.java [hunk 4]
```diff
@@ -192,6 +200,12 @@ public void startSingleEvent(Map<String, String> context) {
               });
       jsonGenerator.writeEndArray();
 
+      // Reset the event namespaces
+      defaultJsonSchemaNamespaceURIResolver.getInstance().resetEventNamespaces();
```
Comment (sboeckelmann) [inline src/main/java/io/openepcis/convert/collector/JsonEPCISEventCollector.java:204]: ...and more ;-)

**Review by sboeckelmann — CHANGES_REQUESTED** [review]

**Comment by Aravinda93** [comment]
@sboeckelmann Fixed the changes requested based on the review. Can you please once verify and approve the PR.

**Review by sboeckelmann — APPROVED** [review]

**Review by sboeckelmann — APPROVED** [review]

**Pull request merged**